src/bridge.js

```diff
@@ -62,3 +123,28 @@ exports.bridge = async function(port, config, registration) {
     bridge.listen(port, config);
     log.info("Matrix-side listening on port:", port);
 };
+
+/**
+ * Returns the email address obtained from Matrix UserId
+ * @param userId        userId in the format `@_email_<localPart>_<domain.tld>:localhost`.
+ * @param homeServer    homeServer address.
+ * @returns {string}    Email-ID obtained from bridge userID `localPart@domain.tld`.
+ */
+function getMailIdFromUserId(userId, homeServer) {
+    const email = userId.slice(8, userId.lastIndexOf(":"+ homeServer));
```

Comment:
When you have magic numbers like `8`, it's not instantly readable as to what you are cutting off. My mentor taught me a nifty trick where you do '@_email_'.length instead of `8`, which gives you the same number but it's obvious what the length is for :)
```suggestion
    const email = userId.slice('@_email_'.length, userId.lastIndexOf(":"+ homeServer));
```